docs/standard/unsafe-code/best-practices.md

```diff
@@ -947,7 +973,7 @@ While most of the suggestions in this document apply to interop scenarios as wel
 
 ## 23. Thread safety
 
-See [Managed threading best practices](../../standard/threading/managed-threading-best-practices.md) and [.NET Memory Model](https://github.com/dotnet/runtime/blob/main/docs/design/specs/Memory-model.md).
+Although memory safety and thread safety are orthogonal concepts, many of the highlighed issues in this document can lead to thread safety issues as well. For example, non-atomic coalesced writes. See [Managed threading best practices](../../standard/threading/managed-threading-best-practices.md) and [.NET Memory Model](https://github.com/dotnet/runtime/blob/main/docs/design/specs/Memory-model.md).
```

Comment:
> many of the highlighed issues in this document

What are the bullet points that this is referring to?

Reply:
I was referring to

10. Memory access coalescing
11. Unaligned memory access

I don't have a strong opinion on this